pkg/contracts/src/CVStrategy.sol

```diff
@@ -75,6 +82,13 @@ library StrategyStruct {
         uint256 maxAmount;
     }
 
+    struct ArbitrableConfig {
+        IArbitrator arbitrator;
+        uint256 collateralAmount;
```

Comment:
I think me and Paul were arguing to have a separated collateral for Challenger and Submitter.
The reason was mainly to let community decide how hard its to dispute proposal without impacting the proposal creation and vis-versa